e2e-tests/test_network_configuration_settings.yml

```diff
@@ -0,0 +1,254 @@
+- name: End-to-End Test for configure_network_settings module
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Apply network configuration changes
+      configure_network_settings:
+        configure_network_type: ovnKubernetes
+      register: patch_result
+
+    - name: Debug applied patch result
+      debug:
+        msg: "{{ patch_result.msg }}"
+
+    - name: Apply network configuration changes
+      configure_network_settings:
+        configure_network_type: ovnKubernetes
+        mtu: 1400
```

Comment:
Why is this hardcoded?  Customers use Jumbo.

Also some OpenShiftSDN customers use MTU < 1280 which prevents IPv6.  We need to check for MTU < 1280.  We have had bugs in the past.

Reply:
This is just a test which checks if the value was set in the config.